howto.md

```diff
@@ -1,14 +1,31 @@
 ---
-title: How to use them
+title: How to Run a Delta Chat Bot
 ---
 
-# How to use
+# How to Run a Delta Chat Bot
+
+To run a bot, you don't need to be a programmer. 
+You don't even need to own or rent a server.
+A bot runs on any computer.
+
+Any bot has a documentation about how to use it - you need to find this information.
+🚧 **TODO**: Where to find the bot's documentation?
+
+## Internet Connection
 
 For running any Delta Chat bot, there is one basic requirement: That a network connection to your email provider is possible via IMAP and SMTP.
 
-Next, checkout the documentation of the bot of your choice. You probably need an installation of the runtime that the bot uses (e.g. Python or NodeJS), but you might already have that.
+## Install the Bot
+
+Next, checkout the documentation of the bot of your choice.
+You need an installation of the runtime (programming language) that the bot uses (e.g. Python or NodeJS), if you don't have that already.
+
+## Every Bot Needs an Email Address
```

Comment:
```suggestion
## Every bot needs an email address
```

Reply:
The title wording doesn't convince me.

Reply:
Headlines are capitalized like that in english: http://www.grammarunderground.com/capitalization-of-headlines.html

Reply:
Ok, but it still does not convince me.